testscript/testscript.go

```diff
@@ -495,6 +495,12 @@ func (ts *TestScript) MkAbs(file string) string {
 	return filepath.Join(ts.cd, file)
 }
 
+// Setenv sets the value of the environment variable named by the key.
+func (ts *TestScript) Setenv(key, value string) {
```

Comment:
Perhaps cmdEnv should also call this function instead of duplicating the logic?
Also, given that we're providing some access to the environment, should we consider providing the capability to get environment variables too?

Reply:
Good call on both fronts. Done